src/solver/conv_ocl_dir2Dfwd_exhaustive_search.cpp

```diff
@@ -602,115 +613,95 @@ ConvOclDirectFwdLegacyExhaustiveSearch::SearchImpl(const ConvolutionContext& par
-                                        min_proc_time = processing_time;
-                                        candidate     = result;
-                                    }
-
-                                    if(run_counter % report_inteval == 0)
+                                    if(solver.IsValidPerformanceConfig(params, result))
                                     {
-                                        MIOPEN_LOG_W("Runs left: " << runs_left << ", "
-                                                                   << "min time so far: "
-                                                                   << min_proc_time
-                                                                   << ", "
-                                                                   << "curr time: "
-                                                                   << processing_time
-                                                                   << ' '
-                                                                   << result);
+                                        auto solution = solver.GetSolution(params, result);
+                                        std::ostringstream os_params;
+                                        result.Serialize(os_params);
+                                        solution.performance_config = os_params.str();
+                                        all_solutions.push_back(solution);
                                     }
-                                    run_counter++;
                                 }
                             }
                         }
                     }
                 }
             }
         }
-    }
-    // Compare search results vs. default performance config.
-    {
-        int ret                   = -1;
-        double default_time       = std::numeric_limits<double>::max();
-        const auto default_config = GetPerformanceConfig(params);
-        if(params.kernel_size_w == 1 && params.kernel_size_h == 1 &&
-           params.group_counts ==
-               1) // Group conv: None 1x1 version yet, fallback to universal kernel.
-        {
-            ret = MeasurePerfConfig<Tgpu, ConvOclDirectFwd1x1>(profile_h,
-                                                               bot_ocl_ptr,
-                                                               top_ocl_ptr,
-                                                               wei_ocl_ptr,
-                                                               bias_ocl_ptr,
-                                                               default_time,
-                                                               params,
-                                                               default_config);
-        }
-        else
+// PrecompileKernels call saves to binary_cache, this needs to be escaped if KERN_CACHE is not on.
+#if MIOPEN_ENABLE_SQLITE_KERN_CACHE
```

Comment:
Ditto

Reply:
@atamazov I'll do it as soon as possible.